packages/shared-ux/chrome/navigation/src/ui/components/navigation_section_ui.tsx

```diff
@@ -6,8 +6,9 @@
  * Side Public License, v 1.
  */
 
-import React, { FC } from 'react';
-
+import React, { FC, useCallback, useEffect, useMemo, useState } from 'react';
+import classNames from 'classnames';
+import { css } from '@emotion/css';
```

Comment:
nit: any reason to use /css instead of /react? https://github.com/elastic/kibana/issues/169355

Reply:
No, I didn't even realized 😄 I guess both work